components/Palettes/tests/unit/UIColor+MDCPalettableTests.m

```diff
@@ -0,0 +1,55 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <XCTest/XCTest.h>
+
+#import "UIColor+MDCPalettable.h"
+
+@interface UIColor_MDCPalettableTests : XCTestCase
+
+@end
+
+@implementation UIColor_MDCPalettableTests
+
+- (void)testBluePalette {
+  // Given
+  MDCPalette *bluePalette = MDCPalette.bluePalette;
+  UIColor *tint500 = bluePalette.tint500;
+  UIColor *accent200 = bluePalette.accent200;
+
+  // Then
+  XCTAssertEqual(tint500.mdc_darkerColorInPalette, bluePalette.tint600);
```

Comment:
Ah. So, here's what's bothering me about `darkerColor...` and `lighterColor...` stuff: It's the NEXT darker and NEXT lighter, right? Just darker or just lighter is inspecific.